developer_tools/Makefile

```diff
@@ -2,14 +2,17 @@ PYTEST_OPTS = --verbose
 
 .PHONY: lint
 lint:
-	./check.py stratis_id 
+	./check.py stratis_id
+	./check.py cargo
 
 .PHONY: fmt
 fmt:
 	isort --recursive check.py stratis_id 
+	isort --recursive check.py cargo
```

Comment:
For isorting, you should specify all the files and directories to be sorted on just one line.